crates/exec-wasmtime/Cargo.toml

```diff
@@ -40,6 +43,9 @@ zeroize = { workspace = true }
 [target.'cfg(windows)'.dependencies]
 io-extras = { workspace = true }
 
+[target.'cfg(unix)'.dependencies]
```

Comment:
Is this cfg necessary? I believe we treat the normal, un-cfg'd dependency list as the "unix" dependencies.

Reply:
I don't get what you mean, this will never be used on anything but Unix, why include it on e.g. Windows?

Reply:
Why is it excluded from Windows in the first place?

Reply:
Because there's no shim on non-unix platforms, it runs within the same binary and logger is only ever initialized once, on the host

Reply:
On Unix platforms, `execute` is used, which will set up a logger for the duration of execution of `exec-wasmtime` and set it back to previous value once execution is done.
On non-Unix platforms, `execute_package` is called directly and, therefore, no logger initialization within `exec-wasmtime` is taking place.
This is working as intended.

https://github.com/enarx/enarx/blob/462c8821a3db8ed1b765c5101c07ced89610950c/src/backend/nil.rs#L105-L115